scripts/setonix-cpu.profile

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+source /opt/cray/pe/cpe/23.03/restore_lmod_system_defaults.sh
+
+module load cmake/3.24.3
+module load cray-mpich
+module load cce/15.0.1
+module load cray-hdf5
+module load cray-python/3.9.13.1
+
+# GPU-aware MPI
+export MPICH_GPU_SUPPORT_ENABLED=0
+
+# allow CMake to find Ascent
+export Ascent_DIR=/software/projects/pawsey0807/bwibking/ascent_06082023/install/ascent-develop/lib/cmake/ascent/
```

Comment:
This Ascent build requires GPU support, so we should get rid of it for the CPU build.